src/components/DropdownInput/DropdownInput.test.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+
+import { DropdownInput } from './DropdownInput';
+
+describe('Test dropdown input component', () => {
```

Comment:
the description of describe block is only the name of the component you are testing, so it is `DropdownInput `